### PR TITLE
routes/application: Add `routeName` tag to all issues

### PR DESCRIPTION
**Pull request by Turbo87**
This will hopefully make it easier to identify which routes have the most issues 😅

### CODE REVIEW

File: app/routes/application.js [hunk 1]
```diff
@@ -2,11 +2,20 @@ import { action } from '@ember/object';
...
 
+import * as Sentry from '@sentry/browser';
+
 export default class ApplicationRoute extends Route {
   @service progress;
+  @service router;
   @service session;
 
   beforeModel() {
+    this.router.on('routeDidChange', () => {
```
Comment (locks) [inline app/routes/application.js:13]: Don't you want to put this in the router itself?

Comment (Turbo87) [reply app/routes/application.js:13]: hmm, I thought extending the router was discouraged these days 🤔

Comment (locks) [reply app/routes/application.js:13]: Hhmm, yeah let's leave it like this.

**Pull request merged**